tachyon_core/benches/db_benchmark_framework.rs

```diff
@@ -265,36 +266,55 @@ impl DatabaseSource for TimescaleDB {
         );
         self.conn.execute(&create_hypertable_query, &[])?;
 
+        let enable_compress_query = format!(
+            "ALTER TABLE {} SET (
+                timescaledb.enable_columnstore = true,
+                timescaledb.segmentby = 'stream_id');",
+            self.table_name,
+        );
+        self.conn.execute(&enable_compress_query, &[])?;
+
+        let move_to_columnstore_query = format!(
+            "CALL add_columnstore_policy('{}', created_before => INTERVAL '1 second');",
+            self.table_name,
+        );
+        self.conn.execute(&move_to_columnstore_query, &[])?;
+
         Ok(())
     }
 
     fn insert(&mut self, timestamps: &[Timestamp], values: &[Value]) -> Result<(), Self::Error> {
         let mut transaction = self.conn.transaction()?;
 
+        let stream_id: i32 = 1;
+
         for (ts, v) in timestamps.iter().zip(values.iter()) {
             match self.value_type {
                 ValueType::Integer64 => {
                     transaction.execute(
-                        &format!("INSERT INTO {} VALUES ($1, $2)", self.table_name),
-                        &[&(*ts as i64), &v.get_integer64()],
+                        &format!("INSERT INTO {} VALUES ($1, $2, $3)", self.table_name),
```

Comment:
One minor nit: I would put stream_id before timestamp and value but this doesn't really matter, just convention